zulip_bots/zulip_bots/simple_lib.py

```diff
@@ -1,10 +1,11 @@
 import configparser
+from contextlib import contextmanager
 import sys
 
-from zulip_bots.lib import BotIdentity
+from zulip_bots.lib import BotIdentity, BotStorage, CachedStorage
 from uuid import uuid4
 
-class SimpleStorage:
+class SimpleStorage(BotStorage):
```

Comment:
I think we don't need to inherit `BotStorage`.

Reply:
Somehow mypy goes wrong here as well if I don't put it here. I'm not really sure why that happens.

Reply:
This is what the error looks like: https://github.com/zulip/python-zulip-api/runs/2590519387?check_suite_focus=true